docs/en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Managing Security Issues
+summary: This document highlights how Silverstripe CMS team handles security issue.
+iconBrand: github-alt
+---
+
+# Managing Security Issues
+
+This document aims to provide a high level overview of how the Silverstripe CMS team handles security issues. Only members of the Silverstripe CMS security team can perform the actions outline in this document.
+
+## Identifying/reporting security issues confidentially 
+
+This process is relevant when a potential vulnerability is reported confidentially and the Silverstripe CMS development team is in a position to prepare a patch prior to the public disclosure of the vulnerability.
+
+This process is usually started once someone [reports a security issue](Issues-and-Bugs/##security-issue).
+
+### When receiving a report
+
+   * An automated response is sent back to the reporter to acknowledge receipt of their vulnerability report.
+   * Perform an initial assessment of the report.
+   * [Create a issue in our private security repository](https://github.com/silverstripe-security/security-issues/issues/new) unless to report is obviously invalid. e.g. SPAM
+   * If encrypted information is provided, attach it to the private security issue.
+   * Reply to [security@silverstripe.org](mailto:security@silverstripe.org) only with a link to the private security issue. Keep most of the discussion on GitHub.
+   * Perform initial criticality assessment. Validate assessment with another member of the security team before replying to the reporter with your conclusion. Ensure the reporter is given a justification for all issues we classify or demote as non-security vulnerabilities. You may need to seek additional information from the reporter before completing the criticality assessment.
+   * Add a new issue in the "Backlog" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
```

Comment:
Sounds like we create a second issue for the same report - probably would be "move the issue to the backlog" instead?